src/media/js/views/search.js

```diff
@@ -117,6 +117,7 @@ define('views/search',
         };
 
         var query = $('#search-q').val() || $('#search-q-desktop').val();
+        var query = query.replace( /  +/g, ' ' );
```

Comment:
Sorry, @waseem18, I was expecting a change to this PR. Go ahead and update and we can move ahead.